.changeset/thick-moles-smash.md

```diff
@@ -0,0 +1,6 @@
+---
+"@vue-flow/background": patch
+"@vue-flow/core": patch
+---
+
+refactor: added vue 2 backwards compatibility for @vue-flow/background
```

Comment:
Refactor are at least `minor` changes (if non-breaking)

Reply:
The package name doesn't have to be repeated in the changeset either, it's already scoped to that packages CHANGELOG file